doc/profile_changes_SLE15.md

```diff
@@ -0,0 +1,473 @@
+      <name>dmz</name>
+      <services config:type="list">
+        <service>ssh</service>
+      </services>
+    </zone>
+    <zone>
+      <name>public</name>
+      <services config:type="list">
+        <service>dhcp</service>
+        <service>dhcpv6</service>
+        <service>samba</service>
+        <service>vnc-server</service>
+      </services>
+    </zone>
+  </zones>
+</firewall>
+```
+
+**The services definition can be added via packages in both cases:**
+
+- https://en.opensuse.org/SuSEfirewall2/Service_Definitions_Added_via_Packages
+- https://en.opensuse.org/Firewalld/RPM_Packaging
+
+Take in account that firewalld already provides most of the more important
+services definitions so check the current services before defining a new one.
+
+
+#### What about the rest of SuSEFirewall2 options?
+
+We would like to continue supporting all the options but unfortunately some
+of them do not have a equivalent mapping in firewalld or need some configuration
+that is still not supported by AutoYaST or by firewalld.
+
+For the options that are not supported by YaST / AutoYaST but are supported by
+firewalld the use of `post-scripts` is probably the best alternative available.
+
+#### Further documentation
+
+- [AutoYaST doc](https://github.com/SUSE/doc-sle/blob/develop/xml/ay_bigfile.xml#L12999)
+- [Firewalld official doc](http://www.firewalld.org/documentation/)
+
+### NTP Configuration
+
+
+The time server synchronization daemon ntpd has been replaced with the more
+modern daemon Chrony.
+
+This change means that AutoYaST files with an ntp_client section need to be
+updated to a new format for this section.
+
+Instead of containing low level configuration options, is now composed of a set
+of high level ones that are applied on top of the default settings.
+
+And here is how the new (and nicer) configuration looks like:
+
+
+```xml
+<ntp-client>
+  <ntp_policy>auto</ntp_policy>
+  <ntp_servers config:type="list">
+    <ntp_server>
+      <iburst config:type="boolean">false</iburst>
+      <address>cz.pool.ntp.org</address>
+      <offline config:type="boolean">true</offline>
+    </ntp_server>
+  </ntp_servers>
+  <ntp_sync>systemd</ntp_sync>
+</ntp-client>
+```
+
+Check out the latest development online documentation for further information
+about each attribute.
+
+- https://susedoc.github.io/doc-sle/develop/SLES-autoyast/html/configuration.html#Configuration.Network.Ntp
+
+### AutoYaST packages are needed for 2nd stage
+
+As you probably already know, a regular installation is performed in a single
+stage while an auto-installation needs two stages in most of the cases.
+
+AutoYaST will show a warning if the second stage is needed or enabled and some
+mandatory package are missing like `autoyast2-installation` and `autoyast2`.
+
+**Further documentation:**
+
+- [AutoYaST doc](https://github.com/SUSE/doc-sle/blob/deb9fe3b4bc13a54c12cc34f56d22b7f31a22db9/xml/ay_bigfile.xml#L139)
+
+
+## Ca Management module has been dropped
```

Comment:
Let's use both letters in uppercase (CA) or maybe full name (Certificate authority)